egs/wsj/s5/steps/nnet3/multilingual/allocate_multilingual_examples.py

```diff
@@ -95,27 +71,18 @@ def get_args():
         'output-2'.""",
         epilog="Called by steps/nnet3/multilingual/combine_egs.sh")
 
-    parser.add_argument("--samples-per-iter", type=int, default=40000,
+    parser.add_argument("--samples-per-iter", type=int, default=10000,
                         help="The target number of egs in each archive of egs, "
```

Comment:
oh... there's one more thing.
It seems that setting --samples-per-iter appropriately is going to be complicated.
I'd prefer if you remove this option and make --num-archives mandatory.
You can just set it, in the calling script, to the sum of the num_archives values in the source directories.